### PR TITLE
Add release notes for 0.18 milestone one

### PR DESCRIPTION
**Pull request by SueChaplain**
Draft release notes to support changes in 0.18 milestone 1.

[ci skip]

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: doc/release-notes/0.18/0.18.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In the previous release, there is a restriction that the <tt>jvmtiSharedCacheInfo.isCorrupt</tt> field and the  <tt>SharedClassCacheInfo.isCacheCorrupt()</tt> method cannot detect a corrupted cache that has a layer number other than zero. This restriction is now removed.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.17 release contains the following known issues and limitations:
```
Comment (PascalSchumacher) [inline doc/release-notes/0.18/0.18.md]: I guess this should be `v0.18`?

Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Absolutely, yes ... always easy to miss one in the source :-) I'll update it in my next commit, but will wait for further review comments before pushing. Thanks for spotting!

File: doc/release-notes/0.18/0.18.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7459">#7549</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: This one is resolved and can be removed.

File: doc/release-notes/0.18/0.18.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+<td valign="top">Startup regression on AIX</td>
+<td valign="top">AIX (64-bit POWER)</td>
+<td valign="top">A 40 - 80% startup regression is under investigation for OpenJDK 8 and 11.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/479">#479</a></td>
+<td valign="top">Non-compressed references VM builds not available</td>
+<td valign="top">AIX, Linux on Z, Linux on Power</td>
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: Linux and Z and Power now have non-compressed refs builds available. Although there is still an open issue at Adopt to add them to the website. https://github.com/AdoptOpenJDK/openjdk-website/issues/627

File: doc/release-notes/0.18/0.18.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+<td valign="top">Non-compressed references VM builds, which support heaps larger than 57GB, are built on Linux on x86, Windows, and macOS. </td>
+<td valign="top">Manual builds on other platforms are possible by following our <a href="https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md">detailed build instructions</a>.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: These last two items about javadoc should be resolved and can be removed. I'll see about closing the issues.

Comment (pshipton) [reply doc/release-notes/0.18/0.18.md]: Actually, they are only resolved for Java 11+.

File: doc/release-notes/0.18/0.18.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9-docs/issues/134">#134</a></td>
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Found #7197

File: doc/release-notes/0.18/0.18.md (hunk 2 above)
Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: +1

File: doc/release-notes/0.18/0.18.md (hunk 4 above)
Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Added (OpenJDK 8 only)

File: doc/release-notes/0.18/0.18.md (hunk 3 above)
Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Updated but did not reference the open issue

File: doc/release-notes/0.18/0.18.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9-docs/issues/7197">#7197</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: The link needs to be changed to reference openj9 instead of openj9-docs

Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Well spotted. Updated

**Comment by pshipton** [comment]
I cancelled the unnecessary travis build.

File: doc/release-notes/0.18/0.18.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.18 release notes
+
+These release notes support the [Eclipse OpenJ9 0.18 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.18/plan).
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: Just noticed this link is incorrect, it should be 0.18.0

File: doc/release-notes/0.18/0.18.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+## Binaries and supported environments
+
+OpenJ9 release 0.18 supports OpenJDK 8, 11, and 13.
+
+Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK 8 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: I wonder if we should update these links to the actual milestone builds when they are available.

File: doc/release-notes/0.18/0.18.md (hunk 7 above)
Comment (pshipton) [reply doc/release-notes/0.18/0.18.md]: Seems it's always been wrong in previous release notes. Luckily the link takes you to https://projects.eclipse.org/projects/technology.openj9/releases where you can follow the links to find the plan.

**Pull request merged**